packages/ui/locales/en/views.json

```
@@ -29,10 +29,19 @@
     "readme": "README.md",
     "openPullReq": "Open pull requests"
   },
+  "createProject": {
+    "newProject": "Create your new project",
+    "description": "Orginaze your projects, pipelines and more.",
```

Spelling and content.

```suggestion
    "description": "Organize your repositories, pipelines and more.",
```